importers/djornl/parser.py

```diff
@@ -144,9 +155,46 @@ def parser_gen(self, file):
                 msg = f"expected {expected_col_count} cols, found {col_count}"
                 yield(line_no, None, f"{file['path']} line {line_no}: {msg}")
 
+    def check_headers(self, headers, validator=None):
+        """
+        Ensure that the file headers contain required columns for the data type. Checks the schema
+        in the validator to ensure that all required fields are present in the headers.
```

Comment:
prevents files from throwing an error every line if the headers are missing a required field